org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -937,6 +972,24 @@ SsoSynapseLlmProdAdmin:
     principalId: !Ref SynapseLlmProdAdminGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-admin-permission-set-arn' ]
 
+SsoSynapseLlmProdDeveloper:
+  Type: update-stacks
+  DependsOn: SsoSynapseLlmDeveloper
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.7/templates/SSO/aws-sso.njk
+  TemplatingContext: {}
+  StackName: !Sub '${resourcePrefix}-${appName}-synapsellm-prod-llmdeveloper'
```

Comment:
should this be `${resourcePrefix}-${appName}-synapsellm-prod-developer`?

Reply:
I wanted to have llmdeveloper vs developer if we have to use a regular developer group later.